meta-ibm/recipes-phosphor/webui/webui-vue/changeLogo.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+model=($(busctl get-property xyz.openbmc_project.Inventory.Manager /xyz/openbmc_project/inventory/system/chassis/motherboard/bmc xyz.openbmc_project.Inventory.Decorator.Asset SubModel))
```

Comment:
The D-Bus object path here is incorrect: `/xyz/openbmc_project/inventory/system/chassis/motherboard/bmc` should be `/xyz/openbmc_project/inventory/system`

Reply:
Done